### ISSUE TITLE
preprocessing and training code for i2v setting

### ISSUE THREAD

**Issue by shengjun-zhang** [XueZeyue/DanceGRPO#25]
Thanks for your excellent work! It inspires me a lot. I notice that the dataset does not support i2v. Did I miss some important information? If the i2v setting is available, would your please tell me which file should I refer to? Thanks a lot!

**Comment by XueZeyue** [comment]
I don't plan to support i2v (SkyReels-I2V) now.
Because this model highly depends on the cfg and many sampling steps (more than 30) to generate reasonable videos. As a result, we need at least 32 H800s to train 3~4 days for just ~100 iterations.
I think these computational resources are infeasible for the community and even for me to tune the open-source hyper-parameters. 
I also think support community-friendly version of SD, FLUX, and HunyuanVideo is enough for the community research.

**Comment by darkliang** [comment]
Thank you for your prompt reply. I am also training GRPO on an internal i2V model and have trained 250 steps with 32 H800s.

<img width="1200" height="600" alt="Image" src="https://github.com/user-attachments/assets/58c01161-4c47-4b82-b886-f26f945c034a" />

 However, I have found that the motion reward has not improved.

**Comment by darkliang** [comment]
Thank you very much for your suggestion. I'll try adjusting the parameters first, as I may not have the resources to train reward model.
